.github/CODEOWNERS

```diff
@@ -330,14 +322,12 @@
 /core/templates/services/guppy-initialization.service.ts @iamprayush
 /core/templates/services/math-interactions.service.ts @iamprayush
 /extensions/domain*.py @brianrodri
-# TODO(#13119): Add @vojtechjelinek as owner after 2021-07-06
-/extensions/interactions/ @iamprayush
```

Comment:
Are we expecting you to add yourself here instead of replacing @iamprayush ? (Ditto in other similar places)

Reply:
Done.